cmd/tink-worker/action.go

```diff
@@ -23,18 +22,35 @@ import (
 var (
 	registry string
 	cli      *client.Client
-	log      *logrus.Entry
+)
+
+const (
+	errImagePull       = "failed to pull image: %s"
+	errCreateContainer = "failed to create container: %v"
+	errRemoveContainer = "failed to remove container: %v"
+	errRunContainer    = "failed to run container"
+	errFailedToWait    = "failed to wait for completion of action: %v"
+	errFailedToRunCmd  = "failed to run on-timeout command: %v"
+
+	infoContainerStatus   = "on %s container status: %v"
+	infoContainerCreated  = "container created: %v"
+	infoContainerRemoved  = "container removed with status: %s"
+	infoWaitFinished      = "wait finished for failed or timeout container"
+	infoExitStatus        = "action container exit status code: %v"
+	infoRemoveContainer   = "starting to remove container: %s"
+	infoStartingContainer = "starting container with cmd: %s"
+	infoRunContainer      = "running container with id: %v"
```

Comment:
Most of these are use only for logging so instead of doing `key: value` this should instead use `With(key, value)` so the dynamic part is logged correctly.

Some of these are also used like `return fmt.Sprintf(...), ...` but there's only one caller to `executeAction` which ignores the returned string anyway. Please remove the string return.